org.openstream.music.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: org.openstream.music
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: openstreammusic
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/openstreammusic/flatpak-node/cache
+        npm_config_cache: /run/build/openstreammusic/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      # Change directory to 'desktop'
+      - rm -r index.html
+      - mv desktop/* .
+      # Install npm dependencies
+      - npm install --offline
+      # Build the app using npx electron-builder
+      - npx electron-builder --linux src --x64
+      # Copy the built files to the Flatpak app directory
+      - cp -a ./build/linux-unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ./run.sh
+    subdir: main
+    sources:
+      - type: archive
+        url: https://github.com/openstreamorg/openstreammusic/archive/refs/tags/v1.2.0.tar.gz
+        sha256: 1107d87a71dcf344aab3d84e9e0abbb82f6439b0af8d580f37002365b705d3bb
+      - type: dir
```

Comment:
Don't use dir type

Reply:
this will not work anyway. there is no `..`